packages/forklift-console-plugin/src/modules/Providers/utils/components/DetailsPage/DetailItem.tsx

```diff
@@ -132,21 +132,24 @@ export const DescriptionTitle: React.FC<{ title: string }> = ({ title }) => (
 );
 
 /**
- * Component for displaying an inline link button with editable content.
+ * Component for displaying an editable content in the following format:
+ * The content field's element and next to that appears a press-able inline
+ * link edit button with the pencil icon, for triggering the onEdit callback.
  *
  * @component
- * @param {ReactNode} content - The content of the button.
- * @param {Function} onEdit - Function to be called when the button is clicked.
+ * @param {ReactNode} content - The field's content element.
+ * @param {Function} onEdit - Function to be called when the button with the pencil icon is clicked.
```

Comment:
```suggestion
 * @param {Function} onEdit - Function to be called when the button is clicked.
```

we only have one button